lstchain/hdu/hdu_table.py

```diff
@@ -0,0 +1,350 @@
+import numpy as np
+import os
+from lstchain.reco.utils import camera_to_altaz
+
+import astropy.units as u
+from astropy.table import Table, Column, vstack, QTable
+from astropy.io import fits
+from astropy.time import Time
+
+__all__ = [
+    'create_obs_hdu_index',
+    'create_event_list'
+    ]
+
+
+DEFAULT_HEADER = fits.Header()
+DEFAULT_HEADER["HDUDOC"] = "https://github.com/open-gamma-ray-astro/gamma-astro-data-formats"
+DEFAULT_HEADER["HDUVERS"] = "0.2"
+DEFAULT_HEADER["HDUCLASS"] = "GADF"
+
+#IRF should be separate?
+def create_obs_hdu_index(filename_list, fits_dir):
+    """
+    Create the obs table and hdu table (below some explanation)
+
+    A two-level index file scheme is used in the IACT community to allow an arbitrary folder structures
+    For each directory tree, two files should be present:
+    **obs-index.fits.gz**
+    (defined in http://gamma-astro-data-formats.readthedocs.io/en/latest/data_storage/obs_index/index.html)
+    **hdu-index.fits.gz**
+    (defined in http://gamma-astro-data-formats.readthedocs.io/en/latest/data_storage/hdu_index/index.html)
+
+    obs-index contains the single run informations e.g.
+    (OBS_ID, RA_PNT, DEC_PNT, ZEN_PNT, ALT_PNT)
+    while hdu-index contains the informations about the locations of the other HDU (header data units)
+    necessary to the analysis e.g. A_eff, E_disp and so on...
+        http://gamma-astro-data-formats.readthedocs.io/en/latest/
+
+    This function will create the necessary data format, starting from the path that contains the DL3
+    converted fits file.
+
+    Parameters
+    ----------
+    filename_list : list
+        list of filenames
+    fits_dir : str
+        directory containing the fits file
+    """
+
+    # empty lists with the quantities we want
```

Comment:
Making a list for each of the required variables is very verbose and makes it hard to follow.

I would recommend to use a single list of dictionaries instead, which makes it most of the time much easier to follow and a lot less code, which is also easier to maintain, because adding something only requires change at a single position, not at two.